pysimple/exceptions.py,sha256=qmHawFszYS905E3fVNciW-81SrYYPFNo6A8E8cFC7GQ,152
pysimple/account.py,sha256=1AJ_ukH3QszwbxjjCfnPvjAW6bexaBfkGEYmDSGGUR4,4602
pysimple/__init__.py,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
pysimple/version.py,sha256=yyeTmR66Y7db_OuiKrX1mvVEt2GmTHE5GlQl-5dcT6A,41
pysimple-0.1.0.dist-info/METADATA,sha256=tRfaMHk4L7hLd-MtQlWFNZ1fBJ7QkSPE-X4YLrF-CXE,1651
pysimple-0.1.0.dist-info/WHEEL,sha256=AvR0WeTpDaxT645bl5FQxUK6NPsTls2ttpcGJg3j1Xg,110
pysimple-0.1.0.dist-info/DESCRIPTION.rst,sha256=OF_9nc7bxE3wOeATnxaI9_A0clEw0gVzd5_LBWcnhgo,807
pysimple-0.1.0.dist-info/top_level.txt,sha256=UgvMsPCcneLRMFfsqDTUCoH_2_llgKo7FS0FiaIZIZ4,9
pysimple-0.1.0.dist-info/metadata.json,sha256=6JwnCitQB8aEiNNaGgHSaeIf5Jap5_pDlvnX0CpR_jA,1024
pysimple-0.1.0.dist-info/RECORD,,
